src/library/filescanner_ffmpeg.c

```diff
@@ -778,3 +802,234 @@ scan_metadata_ffmpeg(struct media_file_info *mfi, const char *file)
 
   return 0;
 }
+
+static int
+file_clone_metadata(AVFormatContext* in_fmt_ctx, const char* dest)
+{
+  int  ret = 0;
```

Comment:
init to sensible values at point of declaration are not useless.  we have to do them later to the same values regardless but as you wish.  Assembly from gcc suggests init and later assign generate the same instructions anyway

codeql reminds us that the `end` label clean up items and relies on init to NULL etc.  otherwise we'll have to add multiple labels and gotos which seems worse than initialise at point of declaration so i've left a commit with those advised by codql